xbmc/network/NetworkFileItemClassify.cpp

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (C) 2005-2020 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "network/NetworkFileItemClassify.h"
+
+#include "FileItem.h"
+#include "utils/URIUtils.h"
+
+namespace KODI::NETWORK
+{
+
+bool IsInternetStream(const CFileItem& item, const bool bStrictCheck /* = false */)
+{
+  if (item.HasProperty("IsHTTPDirectory"))
+    return bStrictCheck;
```

Comment:
Kind of suspicious but it's the existing logic. There isn't any caller with bStrictCheck=true, so the function parameter could be removed.

Reply:
can i do that in a follow-up please?

Reply:
sure, looks good to me otherwise.